packages/engine/src/index.ts

```diff
@@ -261,13 +261,60 @@ class Engine {
 
   @backgroundMethod()
   mnemonicToEntropy(mnemonic: string): Promise<string> {
+    const wordlists = bip39.wordlists.english;
+    const n = wordlists.length;
+    const words = mnemonic.split(' ');
+    let i = 0;
+    while (words.length) {
+      const w = words.pop() ?? '';
```

Comment:
`for (const w of words.reverse()) {` ?

However this method isn't used at all, my suggestion is to remove this completely.